### PR TITLE
Move changelog to a folder and use single file entries

### PR DESCRIPTION
**Pull request by wilzbach**
Single files allow to enforce that every PR comes with a changelog as one can't run into a merge conflicts.

For DMD the idea was to move the language changelog directly to [dlang.org](https://github.com/dlang/dlang.org/pull/1524), s.t. the DMD repo only contains the compiler changes.

For more details, see https://github.com/dlang/tools/pull/186

### CODE REVIEW

**Comment by WalterBright** [comment]
I'll defer this to @MartinNowak

File: changelog/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+How to add a new changelog entry to the pending changelog?
+==========================================================
+
+This will get copied to dlang.org and cleared when master gets
+merged into stable prior to a new release.
+
+```
```
Comment (nemanja-boric-sociomantic) [inline changelog/README.md]: I'm just trying to add the changelog for druntime PR and I find this very confusing. Is what follows the content of the file? If so, what's the name of the file? Maybe add more documentation around here.

File: changelog/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+For more infos, see the [Ddoc spec](https://dlang.org/spec/ddoc.html).
+
+Preview changes
+---------------
+
+If you have cloned the [tools](https://github.com/dlang/tools) and [dlang.org](https://github.com/dlang/dlang.org) repo),
+you can preview the changelog:
+
+```
+../tools/changed.d -o ../dlang.org/changelog/pending.dd && make -C ../dlang.org -f posix.mak html
```
Comment (nemanja-boric-sociomantic) [inline changelog/README.md]: This doesn't seems to work with freshly copied dlang/tools repository:

```
nemanjaboric@labs-129:/home/nemanjaboric/work/druntime  git:(fiber-protection*) $ ../tools/changed.d -o ../dlang.org/changelog/pending.dd && make -C ../dlang.org -f posix.mak html
Usage: ./changed <revision range>, e.g. ./changed v2.067.1..upstream/stable
```

Comment (wilzbach) [inline changelog/README.md]: Yep the downsides of having many repos - PR is still pending: https://github.com/dlang/tools/pull/186

File: changelog/README.md (hunk 1 above)
Comment (wilzbach) [inline changelog/README.md]: > I find this very confusing

Please see https://github.com/dlang/phobos/pull/4228 for the motivation and examples.

> Is what follows the content of the file?

Yes.

> If so, what's the name of the file? 

There's no fixed scheme except that the extension should be `.dd` (Ddoc extension).

> Maybe add more documentation around here.

Ok - Will do so later.

Comment (nemanja-boric-sociomantic) [inline changelog/README.md]: > Please see dlang/phobos#4228 for the motivation and examples.

Sorry, the intention was not confusing, but more how should I use it :-). As being one that wasn't involved in the motivation/examples, I just wanted to say that the `README` is confusing as it doesn't provide clear guidelines what one should do (and there's no prior art in the same repo to rip it off)

File: changelog/README.md (hunk 2 above)
Comment (nemanja-boric-sociomantic) [inline changelog/README.md]: Ah, ok then :+1:

**Comment by wilzbach** [comment]
> I just wanted to say that the README is confusing as it doesn't provide clear guidelines what one should do (and there's no prior art in the same repo to rip it off)

The initial version was a lot more strict and then people demanded to make it more lax :P 
I tried to reformulate the example, s.t. it shows more what one can do with ddoc and I guess once there "is more prior art" it should even be more intuitive.

**Pull request merged**